meson.build

```diff
@@ -0,0 +1,22 @@
+project(
+  'ctre',
+  'cpp',
+  version: '3.9.0',
+  license: 'Apache-2.0',
+  default_options: ['cpp_std=gnu++20,gnu++17,vc++latest,c++20,c++17'],
+)
+
+subdir('include')
+
+ctre_dep = declare_dependency(
+  include_directories: inc,
+  sources: hdrs,
+)
+
+meson.override_dependency('ctre', ctre_dep)
```

Comment:
(If you widen the supported version above, this call requires Meson `>= 0.54`, either globally in `project()` or as a conditional here.)